README.md

```diff
@@ -1,2 +1,34 @@
 Ethereum On-Boarding Testsuite
-==============================
\ No newline at end of file
+==============================
+
+## Implement a Single Node Ethereum Test Network
+
+1. Create an account on https://kurtosistech.com if you don't have one yet.
+2. Verify that the Docker daemon is running on your local machine.
+3. Clone this repository by running `git clone https://github.com/kurtosis-tech/kurtosis-onboarding-experience.git`
+4. Run the empty Ethereum single node test `my_test` to verify that everything works on your local machine.
+   1. Run `bash scripts/build-and-run.sh all`
+   2. Verify that the output of the build-and-run.sh script indicates that one test ran (my_test) and that it passed.
+5. Import the dependencies that are used in this example test suite.
+   1. Run `go get github.com/ethereum/go-ethereum`
+   2. Run `go get github.com/palantir/stacktrace`
+   3. Run `go get github.com/sirupsen/logrus`
+6. Write the Setup() method on the Ethereum single node test in order to bootstrap the network and leave it running and ready to use it.
```

Comment:
I feel like it'd be good to use `### Header`s to group these top-level items, so the checklist isn't nested quite as deep